src/features/pages/PlaySong/components/TopBar.tsx

```diff
@@ -98,3 +105,25 @@ export function ButtonWithTooltip({
     </Tooltip>
   )
 }
+
+type ButtonProps22 = React.InputHTMLAttributes<HTMLInputElement> & {
+  onChangeVolume: (volume: number) => void
+}
+
+export function VerticalSliderVolume({ onChangeVolume, ...rest }: ButtonProps22) {
+  return (
+    <input
+      {...rest}
+      id="default-range"
+      type="range"
+      style={{ WebkitAppearance: 'slider-vertical' }}
```

Comment:
Can you use [appearance](https://developer.mozilla.org/en-US/docs/Web/CSS/appearance) instead of  WebKitAppearance?

Unless you're confident about browser compatibility.